packages/mobile/src/screens/track-screen/TrackRemixesScreen.tsx

```diff
@@ -47,20 +53,38 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
 
 export const TrackRemixesScreen = () => {
   const navigation = useNavigation()
-  const lineup = useSelectorWeb(getRemixesTracksLineup)
-  const count = useSelectorWeb(getCount)
-  const track = useSelectorWeb(getTrack)
-  const user = useSelectorWeb(getUser)
+  const lineup = useSelector(getRemixesTracksLineup)
+  const count = useSelector(getCount)
+  const track = useSelector(getTrack)
+  const user = useSelector(getUser)
+  const dispatch = useDispatch()
 
   const styles = useStyles()
+  const { params } = useRoute<'TrackRemixes'>()
+
+  const trackId = params.id
+  useEffect(() => {
+    dispatch(fetchTrack({ id: trackId }))
+    dispatch(
+      tracksActions.fetchLineupMetadatas(0, 10, false, {
+        trackId: trackId ?? null
+      })
+    )
+  }, [dispatch, trackId])
+
+  useEffect(() => {
+    return function cleanup() {
```

Comment:
Could this be part of the previous useEffect?

Reply:
this is cool!

Reply:
Yeah! I might use this same approach for the profile screen lineup reset that needs to happen